CMakeLists.txt

```diff
@@ -26,6 +27,8 @@ juce_add_plugin(AudioPlugin
     NEEDS_MIDI_OUTPUT FALSE
 )
 
+juce_generate_juce_header(${PROJECT_NAME})
```

Comment:
```suggestion
juce_generate_juce_header(AudioPlugin)
```

Reply:
Not dry, but neither is the `target_compile_features` etc. anyway.

Reply:
Aha I get it now, this is why I got linker errors. Sorry already merged!

Reply:
np! Happy with some movement in this repo 👏